errai-bom/pom.xml

```diff
@@ -193,6 +194,18 @@
         <groupId>com.google.gwt.gwtmockito</groupId>
         <artifactId>gwtmockito</artifactId>
         <version>${version.com.google.gwt.gwtmockito}</version>
+        <!-- Excluding old com.google.gwt 2.8.0 gwt-user and gwt-dev assets
```

Comment:
These exclusions are not necessary if you add the new gwt bom, which will list both newer com.google.gwt and org.gwtproject versions. That newer com.google.gwt version is relocated to org.gwtproject, so no com.google.gwt dependencies will resolve any more.

Reply:
Hi @niloc132, thank you so much for your review, I appreciated it :). 
I imported the new GWT bom, and it worked fine, thank you for suggesting that!
About the version, the main friction that prevents me from using 2.10.1 or 2.11, is related to the gwt maven plugin that doesn't support those versions. I'm aware there's a new plugin that may solve that, but in the legacy projects I'm managing the old plugin is widely used, I would even say nested, so I want to avoid refactoring that.

Reply:
Historically (I haven't used the old plugin in a long time), the old maven plugin could be made happy by managing its dependencies - that is, putting gwt-user and gwt-dev in a `<dependencies>` block within the `<plugin>` itself. Have you tried doing that with the org.gwtproject groupId?

Reply:
I'll take a look, thank you again